releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,21 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.1.0 - November 23, 2023
+
+### Changes
+
+* FeatureManager and ConfigurationFeatureDefinitionProvider are now public.
+   * Enables usage of external dependency injection containers.
+   * Allows usage of feature manager without requiring dependency injection.
+
+* Alternative method `AddScopedFeatureManagement` to register the feature management services.
+   * Enables the registration of the feature manager and feature filters as scoped services.
+
+### Bug Fixes
+* Preserved the behavior in 2.x.x when passing a configuration section to `AddFeatureManagement(configuration)`. ([#308](https://github.com/microsoft/FeatureManagement-Dotnet/issues/308))
```

Comment:
How about we explain what the bug is?

Fixed a bug introduced in the previous release where feature flags cannot be loaded from a custom section of configuration.